src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -42,9 +43,17 @@ def create_connectedk8s(cmd, client, resource_group_name, cluster_name, location
     logger.warning("Ensure that you have the latest helm version installed before proceeding.")
     logger.warning("This operation might take a while...\n")
 
+    #telemetry.add_extension_event('connectedk8s', {'testk': 'testval'})
+    #print(telemetry._session.events['c4395b75-49cc-422c-bc95-c7d51aef5d46'])
+    #return
+
     # Setting subscription id
     subscription_id = get_subscription_id(cmd.cli_ctx)
 
+    # Setting resource Id
+    resource_id = "/subscriptions/{}/resourceGroups/{}/providers/Microsoft.Kubernetes/connectedClusters/{}".format(subscription_id, resource_group_name, cluster_name)
+    telemetry.add_extension_event('connectedk8s', {'Context.Default.AzureCLI.ResourceId': resource_id})
```

Comment:
>add_extension_event [](start = 14, length = 19)

Does this mean a separate event is uped here or this custom prpoerty is added to the event.

Reply:
It will be a new event but will have the same correlation id